azure/run_job.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Again, probably better not to have this as a separate .sh file, just `subprocess.call(kubectl delete -f config/azure-pytorch-elastic.yaml)`  etc. from Python.